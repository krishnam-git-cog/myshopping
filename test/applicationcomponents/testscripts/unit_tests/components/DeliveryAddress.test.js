import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import DeliveryAddress from '../../../../../src/components/deliveryaddress/DeliveryAddress';

describe('DeliveryAddress test', () => {    
    it("Component - DeliveryAddress should match snapshot",  () => {
        let shallow = createShallow();
        const component = shallow(<DeliveryAddress />);
        expect(component).toMatchSnapshot();
      });
});