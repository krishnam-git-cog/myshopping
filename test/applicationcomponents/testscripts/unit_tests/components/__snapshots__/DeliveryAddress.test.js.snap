// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeliveryAddress test Component - DeliveryAddress should match snapshot 1`] = `ShallowWrapper {}`;
